pkgs/tools/networking/openssh/default.nix

```diff
@@ -47,32 +47,24 @@ in
 
   openssh_gssapi = common rec {
     pname = "openssh-with-gssapi";
-    version = "8.4p1";
+    version = "9.0p1";
     extraDesc = " with GSSAPI support";
 
     src = fetchurl {
       url = "mirror://openbsd/OpenSSH/portable/openssh-${version}.tar.gz";
-      sha256 = "091b3pxdlj47scxx6kkf4agkx8c8sdacdxx8m1dw1cby80pd40as";
+      sha256 = "12m2f9czvgmi7akp7xah6y7mrrpi280a3ksk47iwr7hy2q1475q3";
     };
 
     extraPatches = [
-      ./ssh-keysign-8.4.patch
```

Comment:
I think this is the last variant/package using the `ssh-keysign-8.4.patch` (and maybe `ssh-copy-id-fix-eof.patch`) files, so you can go ahead and remove those.